internal/wal_push_handler.go

```diff
@@ -2,21 +2,51 @@ package internal
 
 import (
 	"bytes"
+	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"os"
-	"path/filepath"
 
 	"github.com/pkg/errors"
 	"github.com/spf13/viper"
+	"github.com/wal-g/storages/fs"
 	"github.com/wal-g/tracelog"
 	"github.com/wal-g/wal-g/utility"
+
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
 )
 
 type CantOverwriteWalFileError struct {
 	error
 }
 
+const (
+	WalBulkMetadataLevel       = "BULK"
```

Comment:
Are other our settings capitalized? I'm not sure.

Reply:
I took the example from "WALG_LOG_LEVEL": "NORMAL/DEVEL". Open for any suggestion in-terms of naming covention.